### ISSUE TITLE
Visual Studio Code not working

### ISSUE THREAD

**Issue by uli42** [ArcticaProject/nx-libs#631]
I was just trying to run the current version of Visual Studio Code (https://code.visualstudio.com/; I chose the 64bit tarball) on Centos7 with today's HEAD.

It does not open up but hangs the whole session instead. 

The last lines from xtrace:
```
...
001:<:009d:  4: Request(43): GetInputFocus 
001:>:009d:32: Reply to GetInputFocus: revert-to=PointerRoot(0x01) focus=0x03400026
001:<: Warning: buffer filled!
001:<:009e:8388652: Request(18): ChangeProperty mode=Replace(0x00) window=0x04a00001 property=0x12a("_NET_WM_ICON") type=0x6("CARDINAL") data=0x00000400,0x00000400,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000
000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x
00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x000000
...
0000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000,0x00000000;
001:>:009e:Error 2=Value: major=18, minor=0, bad=6
```

**Comment by uli42** [comment]
See here for some debugging possibilities: https://github.com/Microsoft/vscode/issues/6552

**Comment by Hybrid512** [comment]
Duplicate of this one : https://github.com/ArcticaProject/nx-libs/issues/301
Workaround there : https://github.com/atom/atom/issues/4360

I can confirm that ATOM and VSCode works after the workaround is applied ... it's slow ( as any other Chrome based apps under X2Go) but it works.

**Comment by uli42** [comment]
fixed by #729
